src/duplocloud/args.py

```diff
@@ -261,7 +261,7 @@
            type=bool,
            action='store_true')
 
-WAIT_TIMEOUT = Arg("wait-timeout", "--wait-timeout",
+WAIT_TIMEOUT = Arg("wait-timeout", "--wait-timeout", "--timeout",
```

Comment:
Since `wait-timeout` should have automatically created a `--wait-timeout` flag, but it's declared again here, this duplicate might have been the reason it wasn't registered. I suggest removing it. Alternatively, if the arg was named `waittimeout` which might be more consistent with other args, like `tenantid` that might also fix this issue. Having a shortcut of `--timeout` is good to leave in. It complements `save-secret` having `--save`. 

```suggestion
WAIT_TIMEOUT = Arg("wait-timeout", "--timeout")
```

Reply:
BTW, there are other args declared like this, so if it works here, I'd suggest we take care of those too:

--health-check-url
--container-port
--container-image
--init-container-image
--external-port
--lb-type